src/gamepad/linux/linux-raw-gamepad-monitor.vala

```diff
@@ -64,4 +50,21 @@ private class LibGamepad.LinuxRawGamepadMonitor : Object, RawGamepadMonitor {
 			}
 		}
 	}
+
+	private void initial_device_iterator (GUdev.Device dev) {
+		if (dev.get_device_file () == null)
+			return;
+
+		var identifier = dev.get_device_file ();
+		if ((dev.has_property ("ID_INPUT_JOYSTICK") && dev.get_property ("ID_INPUT_JOYSTICK") == "1") ||
+			(dev.has_property (".INPUT_CLASS") && dev.get_property (".INPUT_CLASS") == "joystick")) {
```

Comment:
The last tab should be 4 spaces as we are aligning the second line of the condition with the first one.